src/components/AppBar/AppBar.js

```diff
@@ -39,6 +40,18 @@ const RVAppBar = props => {
         <Link to={''} className="btn btn-dark btn-block">
           <Button>Home</Button>
         </Link>
+        <Switch
+          defaultChecked
+          value="checked"
+          color="default"
+          onChange={() => {
```

Comment:
I think this will create a new function on each render, regardless if the theme has really changed. I would go with useCallback depending on the value of the currentPalette:

 ```suggestion
  const { classes, currentPalette, setTheme } = props;
  const switchTheme = useCallback(() => {
    if (currentPalette === 'dark') {
      setTheme('light');
    } else {
      setTheme('dark');
    }
  }, [currentPalette]);
```

```suggestion
          onChange={switchTheme}
```